### DISCUSSION TITLE
Software request: webbrowser

### DISCUSSION THREAD

**Discussion by sdsddsd1** [kisslinux/repo#328] [Q&A] [answered]
https://kisslinux.org/answers#005

The current situation to access html from the WWW is, that one have to rely on a huge framework for building firefox. How do you feel about packaging a tiny alternative like lynx or even netsurf-gtk?
Since you stated in `answers` that you want to provide one "modern" webbrowser, a text based one would prob. more suitable.
I think a text based browser would make a good backup plan when something goes terribly  wrong or even gives quick access to web while being in the chroot.

**Comment by dylanaraps** [comment, marked as answer]
I will do an evaluation of many text based browsers and come back with my thoughts. This would also help to make the distribution's website viewable offline.

**Reply by sdsddsd1** [reply]
Btw: there is a maintained fork of elinks. https://github.com/rkd77/elinks

**Comment by jedahan** [comment]
I really like lynx https://lynx.browser.org

screenshot of links on kisslinux.org

<img width="816" alt="Screen Shot 2021-09-02 at 1 07 02 PM" src="https://user-images.githubusercontent.com/32407/131887119-09e7886f-f773-4672-b69f-93198b2b78a2.png">

**Reply by dylanaraps** [reply]
That looks pretty good. I will work on making it render better in the terminal.

**Reply by jedahan** [reply]
Here is regular links (2.23) for comparison. Note that you can remove color in lynx using `-nocolor`
<img width="871" alt="Screen Shot 2021-09-02 at 1 44 43 PM" src="https://user-images.githubusercontent.com/32407/131891909-826498f8-7b28-42d4-8cc3-102befbbc403.png">

Also note that links adds a single char spacing around the site automatically, which is nice.
